subcommand/lifecycle-sidecar/command.go

```diff
@@ -98,22 +111,24 @@ func (c *Command) Run(args []string) int {
 	//
 	// The loop will only exit when the Pod is shut down and we receive a SIGINT.
 	for {
-		cmd := exec.Command(c.flagConsulBinary, c.consulCommand...)
+		start := time.Now()
```

Comment:
Clarifying question: Are we creating a `start` time for solely for debugging or do we intend on utilizing it in for something else as well?

Reply:
This was purely for debugging, I can back it out, but right now it's on the order of 7-10s to run the command (possibly a bug in consul?) with a single registered service, so I was just printing it out.

Reply:
I think it's good to keep this but we could put it in with the existing `failed to sync service` and `successfully synced service` log entries:

```go
		if err != nil {
			logger.Error("failed to sync service", "output", strings.TrimSpace(string(output)), "err", err, "duration", time.Since(start))
		} else {
			logger.Info("successfully synced service", "output", strings.TrimSpace(string(output)), "duration", time.Since(start))
		}
```